projects/sunbird-video-player/src/lib/sunbird-video-player.component.ts

```diff
@@ -217,6 +219,11 @@ export class SunbirdVideoPlayerComponent implements OnInit, AfterViewInit, OnDes
       }
       this.videoInstance.play();
       this.videoInstance.controls(true);
+      // if currently video is not in full screen and was previously full screen then set it back to full screen again
```

Comment:
Can you write the test cases for the same and it should not break existing flows in the portal and mobile apps, Can you confirm same?

Reply:
In the mobile the Interactive Video is played in the landscape mode only so the fullscreen issue does not arise

Reply:
Mobile app planning to implement portrait mode also, So keep that scenario also covered

Reply:
https://github.com/Sunbird-Ed/Community/discussions/15
We previously had a discussion with the mobile team to open Interactive video in landscape mode only

Reply:
Videos (without Interactive content / QuestionSet) can be played in both portrait and landscape mode.
But QuestionSet in the mobile app in portrait is not showing properly so whenever we have 
Interactive video having Questionset in it that is handled my mobile team to play it in landscape mode.
https://github.com/Sunbird-Ed/SunbirdEd-mobile-app/pull/3016